libs/designer-ui/src/lib/picker/pickerItem.tsx

```diff
@@ -13,50 +18,49 @@ const iconInfo = {
       },
     },
   },
-  folder: {
-    iconName: 'FabricFolder',
-    styles: {
-      root: {
-        fontSize: 24,
-        padding: '12px 8px 0 0',
-        transform: 'scale(-1, 1)',
-      },
-    },
-  },
+
   navigate: {
     iconProps: {
       iconName: 'ChevronRight',
     },
   },
 };
 
-const FOLDER_TYPE = 'folder';
+const folderIconStyles: IIconStyles = {
+  root: {
+    fontSize: 24,
+    position: 'relative',
+    bottom: '5px',
+    padding: '0px 8px 0px 0px',
+    transform: 'scale(-1, 1)',
+  },
+};
 
 export interface FileItem {
-  text: string;
-  type: 'folder' | 'file';
+  displayName: string;
+  mediaType: PickerItemType;
+  isParent?: boolean;
   value: any;
-  onSelect?: () => void;
-  onNavigation?: () => void;
+  onSelect?: (selectedObject: any) => void;
+  onNavigation?: (selectedObject: any) => void;
 }
 
 export const PickerItem = (props: FileItem) => {
-  return props.type === FOLDER_TYPE ? <FolderItem {...props} /> : <FileItem {...props} />;
+  return props.mediaType === PickerItemType.FOLDER || props.isParent ? <FolderItem {...props} /> : <FileItem {...props} />;
```

Comment:
mediaType == folder? these normally contain the extension, so if you want to check for folder please only rely on props.isParent, same for line:62